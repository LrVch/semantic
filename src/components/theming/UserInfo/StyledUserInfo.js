import {
  Card
} from 'semantic-ui-react'
import { device } from '../../../media'
import styled from 'styled-components'

const StyledUserInfo = styled(Card)`
  &&& {
    @media ${device.tabletL} {
      margin-bottom: 15px;
    }
  }
`

export default StyledUserInfo