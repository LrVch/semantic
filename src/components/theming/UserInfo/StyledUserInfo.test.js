import 'jest-styled-components'

import React from 'react'
import StyledUserInfo from './StyledUserInfo'
import { device } from '../../../media'
import renderer from 'react-test-renderer'
import styled from 'styled-components'

describe('StyledUserInfo', () => {

  test('should create', () => {
    const tree = renderer.create(<StyledUserInfo />).toJSON()

    expect(tree).toBeTruthy()
  })

  test('should has "margin-bottom" equals to "15px" ', () => {
    const tree = renderer.create(<StyledUserInfo />).toJSON()

    expect(tree).toMatchSnapshot()
    expect(tree).toHaveStyleRule('margin-bottom', '15px', {
      media: `${device.tabletL}`,
      modifier: '&&&'
    })
  })
})


