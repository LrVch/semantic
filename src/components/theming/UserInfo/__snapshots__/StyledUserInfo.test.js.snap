// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledUserInfo should has "margin-bottom" equals to "15px"  1`] = `
@media (max-width:991px) {
  .c0.c0.c0 {
    margin-bottom: 15px;
  }
}

<div
  className="ui card c0"
  onClick={[Function]}
/>
`;
